src/redux/books/books.js

```diff
@@ -2,11 +2,25 @@
 const ADD_BOOK = 'ADD_BOOK';
 const REMOVE_BOOK = 'REMOVE_BOOK';
 
+const defaultState = [
+  {
+    name: 'Harry Potter',
+    author: 'J K Rowling',
+    id: '1',
+  },
+  {
+    name: 'Alice in Wonderland',
+    author: 'Lewis Caroll',
+    id: '2',
+  },
+];
+
 // Reducer
-export default function booksReducer(state = [], action) {
+export default function booksReducer(state = defaultState, action) {
   switch (action.type) {
     case (ADD_BOOK):
-      return [...state].push(action.new);
+      console.log([...state].push(action.new));
```

Comment:
- [ ] I would recommend you remove console log here. It is not a good practice to push console logs in the final iteration of your app. It kills performance and can expose confidential data in more sophisticated apps.